### PR TITLE
Added SUGCON block to Events listing

### PR DESCRIPTION
**Pull request by Trayek**
Added sugcon.md for content
Changed output for eventslist if externalLink is used (title + read more button)

Relates to https://github.com/steviemcg/scuguk/issues/6

### CODE REVIEW

File: src/pages/events/2020/03/sugcon.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+templateKey: 'event-page'
+eventId: 
+title: SUGCON - 26 & 27 March 2020
+sup: The Sitecore User Group Conference (SUGCON) brings the Sitecore user community together and provides the perfect platform to gain knowledge and get inspired by the potential within Sitecore Experience Cloud, Experience Commerce, Content Hub and more. After many very successful events, we are expecting over 500 attendees to attend the event..
+date: 2020-03-26
+dateConfirmed: false
```
Comment (steviemcg) [inline src/pages/events/2020/03/sugcon.md]: Can be true

File: src/pages/events/2020/03/sugcon.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+templateKey: 'event-page'
+eventId: 
+title: SUGCON - 26 & 27 March 2020
+sup: The Sitecore User Group Conference (SUGCON) brings the Sitecore user community together and provides the perfect platform to gain knowledge and get inspired by the potential within Sitecore Experience Cloud, Experience Commerce, Content Hub and more. After many very successful events, we are expecting over 500 attendees to attend the event..
```
Comment (steviemcg) [inline src/pages/events/2020/03/sugcon.md]: potential of

File: src/pages/events/2020/03/sugcon.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+date: 2020-03-26
+dateConfirmed: false
+externalLink: https://europe.sugcon.events/
+venue:
+  name: Budapest Congress Center
+  address: Budapest, Jagelló út 1-3, 1123, Hungary
+  position: 47.4897869, 19.017191507595626
+  details: 
+agenda:
```
Comment (steviemcg) [inline src/pages/events/2020/03/sugcon.md]: Can be removed?

File: src/pages/events/2020/03/sugcon.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+templateKey: 'event-page'
+eventId: 
+title: SUGCON - 26 & 27 March 2020
```
Comment (steviemcg) [inline src/pages/events/2020/03/sugcon.md]: Specify that it's SUGCON **EU**

File: src/pages/events/2020/03/sugcon.md (hunk 3 above)
Comment (Trayek) [reply src/pages/events/2020/03/sugcon.md]: On event listing page the location shows - will remove details and position but I think it's useful to keep address and venue?

File: src/pages/events/2020/03/sugcon.md (hunk 2 above)
Comment (Trayek) [reply src/pages/events/2020/03/sugcon.md]: Not sure what you mean - copied text from Sugcon site?

Comment (steviemcg) [reply src/pages/events/2020/03/sugcon.md]: Hah, it's a grammar nit-pick anyway

File: src/pages/events/2020/03/sugcon.md (hunk 1 above)
Comment (Trayek) [reply src/pages/events/2020/03/sugcon.md]: Had it to false as it's 2 days (26 + 27). Now set to start date/time of session

File: src/pages/events/2020/03/sugcon.md (hunk 3 above)
Comment (steviemcg) [reply src/pages/events/2020/03/sugcon.md]: I have to get used to Github but this was specifically about line 14, Agenda.

File: src/pages/events/2020/03/sugcon.md (hunk 4 above)
Comment (Trayek) [reply src/pages/events/2020/03/sugcon.md]: Done

**Pull request merged**